internal/tsoptions/parsinghelpers.go

```diff
@@ -25,7 +25,10 @@ func parseTristate(value any) core.Tristate {
 
 func parseStringArray(value any) []string {
 	if arr, ok := value.([]any); ok {
-		var result []string
+		if arr == nil {
```

Comment:
Is it possible to have a nil `arr` if it has entered the if block?

Reply:
Yes, `[]any(nil)` is a valid value.